tests/utils/check_code.sh

```diff
@@ -79,7 +79,7 @@ if [[ "$test" == "black" || "$test" == "all" ]]; then
     #       substrings directly)
     echo "CALLING TEST:"
     echo "black --check --extend-exclude _version.py --extend-exclude /lib/ --extend-exclude _docs/ --extend-exclude geoips_dev_utils/ $path"
-    black --check --extend-exclude _version.py --extend-exclude /lib/ --extend-exclude _docs/ --extend-exclude geoips_dev_utils/ $path
+    black --config $path/.config/black $path
```

Comment:
Hmm, this will not work from other repos.  How do we want to handle referencing the geoips repo linter config files from other repos, without having to explicitly reference $GEOIPS_PACKAGES_DIR, or having to check identical config files into every repo?

Reply:
Unfortunately, I think the best option is to check them into every repo. One option might be to rely on something like this: https://github.com/marketplace/actions/repo-file-sync-action.

I really wish there was a way to maintain this kind of thing more easily, but I don't really think there is...

Reply:
@jsolbrig I think we do not treat these as official repo-wide configs, we treat them as reference configs that develiopers can use to configure their linters.  So, we will explictily point to $GEOIPS/.config/black, etc when we run black, etc from our "check_code.sh" script, we can point to that from the .vscode settings iin our vscode repo (since we can assuming geoips repo will be cloned if someone is actively developing), and anyone else can point to them from wherever they want to ensure consistent linting.

Reply:
After discussion - these are just resources.  Let people grab these files and use them to assist in their development if they want.  The "official" test is runnig check_code.sh, which references the latest versions of these.